File: Lecture-3/Code/ArithmeticMean.py
def arithMean(x):
	a = [0] * len(x)
	for i in range(0, len(x)):
		s = 0
		for j in range(0, i+1):
			s = s + x[j]

		a[i] = s / (i+1)

	return a

File: Lecture-3/Code/ArithmeticMean/ArithmeticMean.java
float[] arithMean(float[] x) {
	float[] a = new float[x.length];
	for (int i = 0; i < x.length; i++) {
		float s = 0.0f;
		for(int j = 0; j < i + 1; j++)
			s = s + x[j];

		a[i] = s / (i+1)
	}

	return a;
}